src/services/TensCheckService.ts

```diff
@@ -0,0 +1,27 @@
+interface DataPageResponse {
+  IsNormalAuthentication: boolean;
+  PostAuthAction: string;
+}
+
+async function CheckAuthAndRedirectIfTens(): Promise<void> {
```

Comment:
```suggestion
async function checkAuthAndRedirectIfTens(): Promise<void> {
```

naming convention

Reply:
Can we move this file in helpers please to maintain consistency with common code?